packages/react-native/React/Views/RCTFont.mm

```diff
@@ -423,7 +423,11 @@ + (UIFont *)updateFont:(UIFont *)font
 
   // Handle system font as special case. This ensures that we preserve
   // the specific metrics of the standard system font as closely as possible.
-  if ([familyName isEqual:defaultFontFamily] || [familyName isEqualToString:@"System"]) {
+  if (
+    [familyName isEqual:defaultFontFamily] ||
+    [familyName isEqualToString:@"System"] ||
+    [familyName isEqualToString:@"SystemCondensed"]
+  ) {
```

Comment:
Why is necessary to enter this codepath for SystemCondensed? Do we need to create a new font descriptor here if we already have SystemCondensed?